scripts/doc_checker.py

```diff
@@ -92,8 +93,13 @@ def check_optional_args(doc: Docstring) -> list:
 
     errors = []
     for parameter in optional_args:
+        # case when not all parameters are listed in "Parameters" section
+        if parameter not in doc.doc_parameters:
+            continue
```

Comment:
shouldn't this be an error? or is it already handled by `numpydoc`?

Reply:
It's already handled by `numpydoc`.